webplugin/js/app/conversation/km-conversation-service.js

```
@@ -20,11 +20,18 @@ Kommunicate.conversation = {
             );
             return;
         }
+
         var conversationDetail = data && data.groupFeeds[0];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)